### PR TITLE
Wearable Wet Floor Sign and 'Janitorial Suicide Vest'

### PR DESCRIPTION
**Pull request by Psychpsyo**
## About the PR
This PR ports the tgStation feature where the wet floor signs can also be worn as a vest.
It also slightly updates all the sprites involved to make them more consistent.
Due to the sign now being a vest, the explosive syndicate one now also serves as a janitorial suicide vest.

Note: The new sound ([``Items/Janitor/floor_sign_beep.ogg``](https://github.com/space-wizards/space-station-14/compare/master...Psychpsyo:wet-floor-sign-overhaul?expand=1#diff-4e33b57f6251599f98f442c229842fbd63c4347633ecd16ef7fb67f022ce56de)) is a modified version of [``Machines/Nuke/general_beep.ogg``](https://github.com/space-wizards/space-station-14/blob/master/Resources/Audio/Machines/Nuke/general_beep.ogg) which currently has no attribution so I'm not sure what to put for this new sound or if this can even be used in its current form.

## Why / Balance
The wearable floor sign already existed in tgStation and the explosive vest part was just the logical next step.
As for balance, the sign gives 5% blunt and slash resistance when worn and the explosion stays unchanged so it takes a little over half of your health when you detonate it as a vest.

## Technical details
The prototype WetFloorSign now inherits from ClothingOuterBase and WetFloorSignMineExplosive from WetFloorSign.
The OnUseTimerTrigger component now has a UseVerbInstead boolean that lets it be triggered by a verb in the right click menu instead of the regular interact. (since that already equips the sign as clothing)

## Media
[SS14_sign.webm](https://github.com/space-wizards/space-station-14/assets/60073468/94620811-3e14-4edc-a2db-f15c51df23d3)

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
None, the new boolean on OnUseTimerTriggerComponent defaults to false.

**Changelog**

:cl:
- add: The wet floor sign can now be worn as an outer clothing. This now gives syndicate janitors access to a suicide vest in form the wet floor sign mine.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Specific/Janitorial/janitor.yml [hunk 1]
```diff
@@ -215,15 +219,9 @@
   name: wet floor sign
   suffix: Explosive
   description: Caution! Wet Floor!
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Specific/Janitorial/janitor.yml:221]: you can remove name and desc now

Comment (Psychpsyo) [reply Resources/Prototypes/Entities/Objects/Specific/Janitorial/janitor.yml]: Done.

**Comment by JoeHammad1844** [comment]
make the suicide vest powerful enough to kill anybody within two tiles of you and have it detonate with alt click

**Comment by JoeHammad1844** [comment]
maybe a suicide vest that at least crits people within two tiles away from you could either be a separate uplink item or craftable.. which would require being able to make gas tank bombs like ss13, which would require igniters and assemblies... just a week away

**Comment by Psychpsyo** [comment]
Yea, but I think that should be a dedicated, separate item then.

**Comment by Psychpsyo** [comment]
Also, after going "Ok, how do I make something work on alt-click?" I noticed that the sign already does that just by having the detonate option be highest priority in the right-click menu.

**Pull request merged**